ZichanYang324/BF-Employee-HR
# BGP-43 [Frontend] HR Nav Bar

# Description
Implements BGP-43, HR-Client nav bar.

## Main changes explained:
- Nav bar component.
- Home page route.
…

## How to test:
1. check into current branch
2. install dependencies and run `ng serve`
3. check `/home` route

## Screenshots or videos of changes or expecting result:
![image](https://github.com/ZichanYang324/BF-Employee-HR/assets/85213835/601877d4-ac12-48e9-aec9-b2f0deeae51e)
